src/pages/tBTC/Bridge/Minting/ProvideData.tsx

```diff
@@ -179,17 +192,33 @@ export const ProvideDataComponent: FC<{
         bitcoinNetwork={threshold.tbtc.bitcoinNetwork}
         onSubmitForm={onSubmit}
       />
+      <Checkbox defaultChecked onChange={handleDepositReceiptAgreementChange}>
+        Download Deposit Receipt&nbsp;
+        <Text as="span" color="#888">
+          (Recommended)
+        </Text>
+        <TooltipIcon
+          color="#888"
+          label="The file is extremely important in case you need to make a fast recovery.
+This file is important in the rare case of fund recovery.
+Keep it until you have received your tBTC token. One deposit, one receipt.
+This file contains a BTC recovery address, a wallet public key, a refund public key and a refund lock time of your deposit."
+          ml={3}
+        />
```

Comment:
In fact breaking strings that way is incorrect. The only "breakable" strings in JS are template strings but it results with line breaks in output strings. 
In plain JS/TS such regular string breaks will result as an error. In JSX it's allowed (but not recommended) since it's only a syntactic sugar and it's transpiled to JS though so all the line breaks will be removed by the transpiler. 

The only reasonable way to fit such long strings in 80 characters print width would be to break it into separate strings and concatenate it with addition operator but [it's out of Prettiers scope](https://prettier.io/docs/en/rationale#what-prettier-is-_not_-concerned-about) and also considered as incorrect.

To sum up - such _lenghty_ string values should be treated as an exception in terms of 80 characters print width - just like Prettier does. I applied your request but it needs to be brainstormed - in my opinion (and many others) it's considered incorrect and I had to do it manually.

Ref commit: 93ac29c1